src/main/java/frc/robot/subsystems/intake/IntakeSubsytem.java

```diff
@@ -0,0 +1,100 @@
+package frc.robot.subsystems.intake;
+
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkPIDController;
+import com.revrobotics.CANSparkBase.ControlType;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.PortConstants;
+
+public class IntakeSubsytem extends SubsystemBase{
+
+    private final CANSparkMax leaderMotor;
+    private final CANSparkMax followerMotor;
+    private final CANSparkMax pivotMotor;
+
+
+    private final RelativeEncoder leaderEncoder;
+    private final RelativeEncoder followerEncoder;
+    private final RelativeEncoder pivotEncoder;
+
+    private final DigitalInput intakeLimitSwitch;
+
+    private final SparkPIDController pivotPIDController;
+
+
+    public IntakeSubsytem(){
+        leaderMotor = new CANSparkMax(PortConstants.LEADER_INTAKE_MOTOR_ID, MotorType.kBrushless);
+        followerMotor = new CANSparkMax(PortConstants.FOLLOWER_INTAKE_MOTOR_ID, MotorType.kBrushless);
+        followerMotor.setInverted(true);
```

Comment:
This might not be necessary - maybe comment it out but leave it there in case we do need it